style.css

```diff
@@ -0,0 +1,309 @@
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+  }
+  header nav {
+    padding-top: 40px;
+    margin-right: 50px;
+    float: right;
+    font-family: "Lucida Sans", "Lucida Sans Regular", "Lucida Grande",
+      "Lucida Sans Unicode", Geneva, Verdana, sans-serif;
+    display: flex;
+    font-variant: small-caps;
+    font-size: 25px;
+    font-weight: bold;
+    border-top-color: transparent;
+  }
+  header li:hover {
+    transform: scale(1.1);
+    font-weight: bolder;
+  }
+  header li {
+    list-style-type: none;
+    display: inline-block;
+    margin-left: 25px;
+  }
+  #about {
+    margin-bottom: 15%;
+  }
+  #about p {
+    font-size: 25px;
+    line-height: 1cm;
+  }
+  
+  a {
+    text-decoration: none;
+    color: black;
+  }
+  body {
+    background-image: url("./images/bg.jpg");
```

Comment:
Choose a different background image for your own portfolio ;-)
I like this site
https://www.svgbackgrounds.com/